pkg/grid/src/nav/Help.tsx

```diff
@@ -1,34 +1,114 @@
 import React from 'react';
-import helpAndSupport from '../assets/help-and-support.svg';
+import cn from 'classnames';
+import { Button } from '../components/Button';
+
+interface Group {
+  title: string;
+  icon: string;
+  color: string;
+  link: string;
+}
+
+const groups: Record<string, Group> = {
+  uc: {
+    title: 'Urbit Community',
+    icon: 'https://fabled-faster.nyc3.digitaloceanspaces.com/fabled-faster/2021.4.02..21.52.41-UC.png',
+    color: 'bg-black',
+    link: '/apps/landscape/~landscape/ship/~bitbet-bolbel/urbit-community'
+  },
+  discovery: {
+    title: 'Group Discovery',
+    icon: 'https://urbit.me/images/icons/icon-512x512.png',
+    color: 'bg-green-300',
+    link: '/apps/landscape/~landscape/ship/~rondev/group-discovery'
+  },
+  foundation: {
+    title: 'Foundation',
+    icon: 'https://interstellar.nyc3.digitaloceanspaces.com/battus-datsun/2022.6.10..21.28.21-urbit-inverted.png',
+    color: 'bg-black',
+    link: '/apps/landscape/~landscape/ship/~wolref-podlex/foundation'
+  },
+  forge: {
+    title: 'The Forge',
+    icon: '',
+    color: 'bg-black',
+    link: '/apps/landscape/~landscape/ship/~middev/the-forge'
+  },
+  tlonSupport: {
+    title: 'Tlon Support Forum',
+    icon: '',
+    color: 'bg-yellow-500',
+    link: '/apps/landscape/~landscape/ship/~bitpyx-dildus/tlon-support'
+  }
+};
+
+const GroupLink = ({ group }: { group: Group }) => (
+  <div className="flex justify-between items-center py-2">
+    <div className="flex space-x-2">
```

Comment:
I think you want items-center here so that the group names are centered with the avatar